### DISCUSSION TITLE
[Bug]: Back Button Inconsistent Behavior

### DISCUSSION THREAD

**Discussion by mrmorganmurphy** [blakeblackshear/frigate#12147] [Report a Bug] [answered]
### Describe the problem you are having

In the 0.14beta 3 the back button in review mode does not behave as expected when playback speed has been adjusted.
Instead of going back to the snapshot view, or the camera selection, it re-adjusts the playback speed to what it was previously.
This behavior makes the PWA feel slow and unresponsive as it's not immediately clear what is happening.

### Written Steps
1. Navigate to a stream from within the review area.
2. Adjust the playback speed from the default
3. In this case, adjusted from 1x to 8x 
![image](https://github.com/blakeblackshear/frigate/assets/22598031/28313030-3600-462a-aedd-792197bfcda1)
4. Press the back button
5. Playback speed reverts to prior choice
6. Video speed does not adjust
7. Press back button again
8. UI returns to camera/clip selection

If the speed is adjusted multiple times, the time goes back in increments. So if you adjust the speed 2x you have to press the back button 3x to return to the camera/clip selection screen.
### Visual of the Issue
![Frigate_UI_Issue](https://github.com/blakeblackshear/frigate/assets/22598031/1eb93328-cc72-4b64-b777-898246b37fd0)

### Version

0.14.0-4e5a6eb

### Frigate config file

```yaml
Not relevant.
```

### Relevant log output

```shell
No relevant log output.
```

### Operating system

Proxmox

### Install method

Docker Compose

### Network connection

Wired

### Camera make and model

Unifi G5 Bullet & G3 Instant

**Comment by NickM-27** [comment, marked as answer]
Fixed in https://github.com/blakeblackshear/frigate/pull/12153
